.github/workflows/release-python.yml

```diff
@@ -159,11 +159,12 @@ jobs:
       with:
         name: all-dist-${{ github.run_id }}
         path: dist/
-    - name: Publish distribution 📦 to PyPI
-      if: startsWith(github.ref, 'refs/tags/')
-      uses: pypa/gh-action-pypi-publish@release/v1
     - name: Publish package distributions to TestPyPI
       uses: pypa/gh-action-pypi-publish@release/v1
       with:
         repository-url: https://test.pypi.org/legacy/
         skip-existing: true
+        attestations: ${{ !startsWith(github.ref, 'refs/tags/') }}
```

Comment:
besides this line, what's changing?

Reply:
That's it, the previous one errored out because the attestations already existed for the dist files.

Reply:
And I inverted the order to run testpypi first